AzureFunctions.v3.Durable/Orchestrator.cs

```diff
@@ -14,7 +16,8 @@ public static class Orchestrator
         [FunctionName("Orchestrator")]
         public static async System.Threading.Tasks.Task<HttpResponseMessage> RunAsync([HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequestMessage req, [DurableClient] IDurableClient starter, ILogger log)
         {
-            var events = await req.Content.ReadAsAsync<EventGridEvent[]>();
+            var reader = await req.Content.ReadAsStringAsync();
+            var events = EventGridEvent.ParseMany(BinaryData.FromString(reader));
```

Comment:
We can use Parse and remove the next line that does SingleOrDefault which would throw if there is more than one element anyway.